cmd/snap-bootstrap/cmd_initramfs_mounts_test.go

```diff
@@ -3740,8 +3723,10 @@ grade=signed
 	c.Assert(filepath.Join(dirs.SnapBootstrapRunDir, fmt.Sprintf("%s-model-measured", s.sysLabel)), testutil.FilePresent)
 }
 
-func (s *initramfsMountsSuite) TestInitramfsMountsRecoverModeDegradedUnencryptedDataSaveEncryptedUnhappy(c *C) {
-	// test a scenario when data is unencrypted but save is encrypted
+func (s *initramfsMountsSuite) TestInitramfsMountsRecoverModeDegradedUnencryptedDataSaveEncryptedHappy(c *C) {
```

Comment:
this test setup now seems to duplicate the test I mentioned above that needs to be renamed (`TestInitramfsMountsRecoverModeDegradedAbsentDataSaveFallbackHappy` in the current state of the pr) ?

Reply:
I guess in retrospect this test is okay, as this one has the ubuntu-save-enc in the mock disk while the other one doesn't, but the one below definitely does seem to duplicate TestInitramfsMountsRecoverModeDegradedAbsentDataSaveFallbackHappy

Reply:
I think those are slightly different still. The test above checks a scenario with no data, but unencrypted save. The one here tries a case with unencrypted data, but encrypted save. Finally, the one below (TestInitramfsMountsRecoverModeDegradedEncryptedDataUnencryptedSaveHappy) tries encrypted data, but unencrypted save. Though with all the mocking all 3 appear to be walls of text basically.